azure-pipelines/run_test_steps.yml

```diff
@@ -53,6 +53,8 @@ steps:
     continueOnError: ${{ parameters.allowFailure }}
     env:
       TWISTED_REACTOR: ${{ parameters.windowsReactor }}
+      PYTHONUTF8: "1"
+      PYTHONIOENCODING: "utf8:surrogateescape"
```

Comment:
It looks like this was an attempt to fix the failing windows tests… which are still failing.

In general I am concerned about adding env vars like this to the test suite, since we might inadvertently introduce features into Twisted where it *can't* work without custom configuration on Windows, so we should remove this.